StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1605PartialElementDocumentationMustHaveSummary.cs

```diff
@@ -66,6 +66,7 @@
     /// SDK documentation tools.</para>
     /// </remarks>
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    [NoCodeFix("Cannot generate documentation")]
```

Comment:
:question: This is an interesting case. If the `<summary>` (or `<content>`) element is missing altogether, we could generate an empty one even if the result would still produce SA1605 due to the element being empty. Do we (or might we in the future) want to provide this functionality?